spec/spec_helper.rb

```diff
@@ -21,7 +22,7 @@
   config.filter_run_when_matching :focus
 
   def app
-    Rack::Lint.new(Rack::Test::FakeApp.new)
```

Comment:
I don't know how you feel about this, but I prefer if this can be an object we instantiate rather than a singleton.

Reply:
Agreed, I made that change.